sdk/js/jest.config.json

```diff
@@ -5,5 +5,5 @@
   "testRegex": "(/__tests__/.*|(\\.|/)(test|spec))\\.(jsx?|tsx?)$",
   "testPathIgnorePatterns": ["__tests__/utils"],
   "moduleFileExtensions": ["ts", "tsx", "js", "jsx", "json", "node"],
-  "testTimeout": 60000
+  "testTimeout": 300000
```

Comment:
the manual jest timeout settings i saw were 120k, this change takes timeout from 60k->300k

just picked a big number or experimentally found to be long enough?

Reply:
Heh, basically. Some of the tests with conflicting resources caused the timeouts to grow slightly higher (which triggered the last change to 120) and this switch to Anvil seemed to take some just over that limit. I would hope this could be resolved with some better / hard-coded resource allocation, but that will be quite a slog to get right. `300k` just felt slightly better than doubling it to `240k` :shrug: